docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -1757,7 +1757,7 @@ a base64-encoded AuthConfig object.
 
         ```
     {
-            "registrytoken": "9cbaf023786cd7..."
+            "identitytoken": "9cbaf023786cd7..."
```

Comment:
Was the change versioned, or is it now changed for all versions? (depending on the version of the daemon)